src/Utilities/FlowAnalysis/FlowAnalysis.Utilities.projitems

```diff
@@ -145,6 +145,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)FlowAnalysis\Analysis\TaintedDataAnalysis\HardcodedCertificateSinks.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FlowAnalysis\Analysis\TaintedDataAnalysis\HardcodedCertificateSources.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FlowAnalysis\Analysis\TaintedDataAnalysis\HardcodedBytesSources.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)FlowAnalysis\Analysis\TaintedDataAnalysis\HardcodedSymmetricAlgoritthmKeysSources.cs" />
```

Comment:
Nit: the file name has a typo (double 't')

Reply:
Oops!